Dockerfile

```diff
@@ -12,12 +12,15 @@ RUN cmake -DCMAKE_BUILD_TYPE=Release .. && make -j $(nproc) && make test
 
 FROM base as runtime
 WORKDIR /app
+RUN apt-get update && apt-get install -y wget python3-yaml unzip curl
 ARG UID=1000
 RUN groupadd -r qlever && useradd --no-log-init -r -u $UID -g qlever qlever && chown qlever:qlever /app
-USER qlever
-ENV PATH=/app/:$PATH
+
 COPY --from=builder /app/build/*Main /app/src/web/* /app/
+COPY --from=builder /app/e2e/* /app/e2e/
+ENV PATH=/app/:$PATH
 
+USER qlever
 EXPOSE 7001
 VOLUME ["/input", "/index"]
```

Comment:
Would it help or make a difference to add `" /e2e_data"` to the volumes? I think not having to download the zip everytime into the internal storage would be nice. I think then this should also be mentioned in the documentation then.

Reply:
Well in the previous version we used Travis' caching but that just means we download from one of their servers. But yeah might be a bit more civil. I'll look into that for Travis. Though since I just saw that the docker cloud thing also supports GitHub's Checks API I'm not sure we'll keep that around.